applications/serial_lte_modem/doc/TCPUDP_AT_commands.rst

```diff
@@ -174,7 +174,7 @@ Syntax
 
 ::
 
-   #XTCPCLI=<op>[,<url>,<port>[,sec_tag[,peer_verify[,hostname_verify]]]]
+   #XTCPCLI=<op>[,<url>,<port>[,sec_tag[,peer_verify[,hostname_verify,[cid]]]]]
```

Comment:
Is this PR trying to change all parameters from sec_tag onwards so that they can be empty as well?
So it would be like:
```suggestion
   #XTCPCLI=<op>[,<url>,<port>[,[sec_tag][,[peer_verify][,[hostname_verify][,[cid]]]]]]
```

Reply:
Both of the proposals look a little bit unreadable. This syntax is not supposed to be syntactically correct, but more like a human readable.

Look example from [AT%KEYGEN](https://docs.nordicsemi.com/bundle/ref_at_commands/page/REF/at_commands/security/keygen_set.html)

All options are shown as `[,option]` but some are described to be optional, so they could be left out.